apps/home/src/components/LocationSelector/styles/LocationSelector.module.scss

```
@@ -0,0 +1,25 @@
+@use "@carbon/react/scss/spacing" as *;
+@use "@carbon/react/scss/type" as *;
+
+.dropdownWrapper {
+  :global(.cds--dropdown) {
+    min-width: 12rem;
+  }
+}
+
+.loading {
+  @include type-style("label-01");
+  padding: $spacing-03 $spacing-04;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)